### PR TITLE
Feature/add sections

### PR DESCRIPTION
**Pull request by pm-moyanor**
hey, added basic sections, check if works for you, and let me know if you want more help, looks like a fun project

### CODE REVIEW

**Comment by Akshat2Jain** [comment]
Can You show these components on the homepage? I want to see a preview of how they look

File: client/src/components/Testimonial.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styles from "../styles/Testimonials.css"
```
Comment (Akshat2Jain) [inline client/src/components/Testimonial.jsx]: Looks Like something is wrong with this statement.

Comment (pm-moyanor) [reply client/src/components/Testimonial.jsx]: I'll take a look at it

**Comment by Akshat2Jain** [comment]
Looks Good! Thanks

**Pull request merged**